app/views/supervisor/dashboard/index.html.slim

```diff
@@ -1,16 +1,29 @@
 h1 = "Hi #{current_user.name}!"
-h2 This is your brand-new Supervisor Dashboard - with love from Team Cheesy
+h2 Welcome @ your Supervisor Dashboard
 p alpha version
 
-hr
-
 div.clearfix
   - supervised_teams.each do |t|
-      div.dashboard-box
-        h3 = link_to t.name, t
-        - if t.last_checked_at?
-          p = "Your latest check was #{ format_date(t.last_checked_at, :short) }"
+    div.dashboard-box
+      section.team-title
+        h3 = link_to "Team " + t.name, t
+      section.dashboard
+        h4 Checks
+        - if t.comments.any?
+          p = "Your latest check was #{ time_ago_in_words(t.comments.first.created_at) } ago."
+        - else
+          p You didn't check upon them yet.
+        - if can?(:supervise, t)
```

Comment:
Do we need this check? supervised_teams already selects the teams their are supervising.